config/default/kustomization.yaml

```diff
@@ -18,6 +18,10 @@ bases:
   - ../server
   - ../server-rbac
 
-patchesStrategicMerge:
-  - manager_image.yaml
-  - server_image.yaml
+images: 
+- name: tink-controller
+  newName: TINK_CONTROLLER_IMAGE
+  newTag: TINK_CONTROLLER_TAG
+- name: tink-server
+  newName: TINK_SERVER_IMAGE
+  newTag: TINK_SERVER_TAG
```

Comment:
looks like some yamllint formatting will be required here.

Reply:
removed this section so this shouldn't be an issue anymore